com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -96,6 +96,12 @@ public class NetworkSceneManager
         /// </summary>
         private static bool s_IsSceneEventActive = false;
 
+        // TODO: Remove `m_IsRunningUnitTest` entirely after we switch to multi-process testing
+        // In MultiInstance tests, we cannot allow clients to load additional scenes as they're sharing the same scene space / Unity instance.
```

Comment:
in the previous PR #1030, @andrews-unity and @LukeStampfli approved this PR and I also do approve this PR too.
please feel free to approve your own PR @NoelStephensUnity and merge it in :)